### PR TITLE
wrapping management api

### PR DESCRIPTION
**Pull request by BetaCat0**

### CODE REVIEW

File: proto/casbin.proto [hunk 1]
```diff
@@ -97,17 +104,14 @@ message EmptyReply {
...
+  Policy policy = 3;
 }
 
-message NamedPolicyRequest {
-  int32 enforcerHandler = 1;
-  string ptype = 2;
-  string sub = 3;
-  string obj = 4;
-  string act = 5;
+message Policy {
```
Comment (hsluoyz) [inline proto/casbin.proto]: ``Policy`` includes the ``ptype``

**Review by hsluoyz — CHANGES_REQUESTED** [review]
Please fix.

File: proto/casbin.proto [hunk 2]
```diff
@@ -126,6 +124,14 @@ message GetPolicyRequest {
   repeated string fieldValues = 4;
 }
 
+message RemovePolicyRequest {
+  int32 enforcerHandler = 1;
+  string pType = 2;
+  int32 fieldIndex = 3;
+  repeated string fieldValues = 4;
+  repeated string params = 5;
```
Comment (hsluoyz) [inline proto/casbin.proto]: ``params`` should not be here. You can see the function code, it's not used. And ``RemovePolicyRequest()`` can be merged into ``GetPolicyRequest()``.

**Review by hsluoyz — CHANGES_REQUESTED** [review]
Please fix.

File: proto/casbin.proto (hunk 2 above)
Comment (BetaCat0) [reply proto/casbin.proto]: I think we can merge RemovePolicyRequest, GetPolicyRequest, NamedPolicyRequest and PolicyRequest itself into one, maybe named  `PolicyRequest` ?

Comment (hsluoyz) [reply proto/casbin.proto]: Just keep the same as the Casbin's API

Comment (hsluoyz) [reply proto/casbin.proto]: ``NamedPolicyRequest()`` has the ``ptype`` param.

**Review by hsluoyz — APPROVED** [review]

**Pull request merged**